.ddev/.env.sample

```diff
@@ -0,0 +1,27 @@
+# SAFE ENV VARS, RELEVANT FOR LOCAL DEVELOPMENT.
+CONFIG_READONLY=0
+DB_HOST=db
+DB_USER=db
+DB_PASS=db
+DB_NAME=db
+DB_PREFIX=
+DB_PORT=3306
+DB_NAMESPACE=Drupal\Core\Database\Driver\mysql
+DB_DRIVER=mysql
+SIMPLEI_ENV_COLOUR=#637C16
+SIMPLEI_ENV_NAME=Local
+
+# SENSITIVE ENV VARS. KEYS ONLY.
+CONFIG_SYNC_DIRECTORY=/var/www/html/project/config
+FILE_PRIVATE_PATH=/private
+HASH_SALT=
+NW_TEST_USER_PREFIX=
+TEST_USER=
+TEST_PASS=
+TEST_D7_URL=
+STAGE_FILE_PROXY_ORIGIN=
+
+# SENSITIVE PLATFORM.SH
+PLATFORM_PROJECT=kamkoebtv74zm
+PLATFORM_ENVIRONMENT=main
+PLATFORMSH_CLI_TOKEN=<add your token here, it should be a personal API token retrieved from the Platform.sh dashboard>
```

Comment:
I think some files are out of sync. I don't recommend adding the CLI token to the env file anymore in case it gets accidentally published. The hosting repo contains an updated env sample: https://github.com/dof-dss/maestro-hosting/blob/MAES-3/resources/unity/DDev/templates/.env.sample

Reply:
Yes, saw that but I couldn't get it to work without putting the token directly in this .env file - I'll have another go

Reply:
Got that to work now, have done as you suggested